app/api/groups/get_group_progress.feature

```diff
@@ -222,6 +222,7 @@ Feature: Display the current progress of a group on a subset of items (groupGrou
       | 0          | 15             | 211     | 2017-04-29 06:38:38 | 0              | null                | 0            | 0           | null                |
       | 0          | 15             | 212     | 2017-03-29 06:38:38 | 0              | null                | 0            | 0           | null                |
       | 0          | 59             | 212     | 2019-01-01 00:00:00 | 10             | null                | 0            | 0           | null                |
+      | 0          | 59             | 214     | 3019-01-01 00:00:00 | 0              | null                | 0            | 0           | null                |
```

Comment:
your test do not really check the output is not <0

Reply:
That's not true. I wrote the test first.